nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -1476,12 +1486,14 @@ public String toString() {
         private final int dataType;
         private final boolean required;
         private final Integer columnSize;
+        private final boolean isNullable;
```

Comment:
Minor naming recommendation, but following Java Bean conventions, what do you think about renaming this field to `nullable`?  The accessor method would keep the existing name of `isNullable`.

Reply:
Yeah good call, will change